test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -792,6 +792,22 @@ public void editFile(String filename) {
         }
     }
     
+    public void editFile(String filename, String comment) {
+        
+        System.err.println("editFile: "+comment+" file: "+filename);
```

Comment:
I thought we agreed that Test methods shouldn't output to stdout/stderr?   (Certainly we have a LOT of places that already do... but why are we adding more?)